### PR TITLE
Fix point 5.6, ref discussions/14 🔨

### PR DESCRIPTION
**Pull request by alivx**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -133,6 +133,12 @@ account_inactive: 30
 ## 5.4.5 Ensure default user shell timeout is 900 seconds or less
 shell_timeout_sec: 900
 
+# Section 5 PAM config
+usersAllowedToAccess:
+  - root
+  - ubuntu
```
Comment (jvleminc) [inline defaults/main.yml:139]: I'd remove user "ubuntu" here. He can always "su" through "sudo su".

File: defaults/main.yml [hunk 2]
```diff
@@ -133,6 +133,12 @@ account_inactive: 30
 ## 5.4.5 Ensure default user shell timeout is 900 seconds or less
 shell_timeout_sec: 900
 
+# Section 5 PAM config
+usersAllowedToAccess:
```
Comment (jvleminc) [inline defaults/main.yml:137]: Better: usersAllowedToAccess => usersAllowedTo**Su**Access ?

File: tasks/section_5_Access_Authentication_and_Authorization.yaml [hunk 3]
```diff
@@ -758,11 +758,23 @@
...
-    groups: ["sudo"]
+  block:
+    - name: 5.6 Ensure access to the su command is restricted  | create su-group-access
+      template:
+        src: files/templates/etc/su-group-access.j2
+        dest: "/etc/security/su-group-access"
+        owner: root
+        group: root
+        mode: 0644
+    - name: 5.6 Ensure access to the su command is restricted  | create su-group-access
```
Comment (jvleminc) [inline tasks/section_5_Access_Authentication_and_Authorization.yaml:769]: *...create /etc/pam.d/su

**Review by jvleminc — CHANGES_REQUESTED** [review]

File: tasks/section_5_Access_Authentication_and_Authorization.yaml (hunk 3 above)
Comment (alivx) [reply tasks/section_5_Access_Authentication_and_Authorization.yaml:769]: What is the issue here?

Comment (jvleminc) [reply tasks/section_5_Access_Authentication_and_Authorization.yaml:769]: You copypasted from  above, here the line should be:
```
- name: 5.6 Ensure access to the su command is restricted  | create /etc/pam.d/su
```

Comment (alivx) [reply tasks/section_5_Access_Authentication_and_Authorization.yaml:769]: Lol, ok

**Pull request merged**